pyscriptjs/src/plugin.ts

```diff
@@ -85,10 +100,47 @@ export class PluginManager {
     afterStartup(runtime: Runtime) {
         for (const p of this._plugins)
             p.afterStartup(runtime);
+        for (const p of this._pythonPlugins)
+            if (typeof p.afterStartup !== 'undefined') {
+                p.afterStartup(runtime);
+            }
     }
 
     onUserError(error: UserError) {
         for (const p of this._plugins)
             p.onUserError(error);
     }
 }
+
+
+export function create_custom_element(name: string, pyObject: any, callback?: Function) : any {
+    logger.info(`creating plugin: ${name}`)
+    class CustomElementWrapper extends HTMLElement {
+        shadow: ShadowRoot;
+        wrapper: HTMLElement;
+        pyObject: any;
+        source: string;
```

Comment:
Yeah... it is hard :) So, I've changed:

* `create_custom_element` to `define_custom_element`
* `pyObject` to `pyPluginClass` and `this.pyObject` to `this.pyPluginInstance` (even if, thinking a bit more, these don't **need** to be a Python class/instance, can be anything really)
* `CustomElementWrapper` to `ProxyCustomElement` (even if I'm still not that happy...). I didn't think `CustomElement` or `MyCustomElement` were great because it's not conveying the fact that it's really an Proxy Custom Element that delegates logic to `this.pyPluginInstance`. 
* `source` to `originalInnerHTML`.  `innerHTML` isn't great because it actually clashes with the `HTMLElement.innerHTML` attribute and it'd also give the idea that it's the current `innerHTML` while it's really meant to be the original `innerHTML` (the reason it was named was because I started with the `py-script` tag and that coincided with the source code within the tag but that's not true for all Custom Elements anymore)